ci/linux-packages/debian/rules

```diff
@@ -31,17 +31,19 @@ export DEB_BUILD_MAINT_OPTIONS=reproducible=-timeless
 CMAKE_BUILD_TYPE = RelWithDebInfo
 
 override_dh_auto_configure:
-	dh_auto_configure				\
-	  --sourcedirectory=c/		                \
-	  --builddirectory=c.build               	\
-	  --buildsystem=cmake+ninja			\
-	  --						\
-	  -DCMAKE_BUILD_TYPE=$(CMAKE_BUILD_TYPE)        \
-          -DADBC_DRIVER_MANAGER=ON                      \
-          -DADBC_DRIVER_POSTGRESQL=ON                   \
-          -DADBC_DRIVER_SQLITE=ON                       \
-          -DADBC_DRIVER_FLIGHTSQL=ON                    \
-          -DADBC_DRIVER_SNOWFLAKE=ON
+	env \
+	  PATH=$(PATH):/usr/lib/go-1.20/bin \
```

Comment:
I think that environment variable not Makefile variable is better here:

```suggestion
	  PATH=$${PATH}:/usr/lib/go-1.20/bin			\
```

(`PATH=/usr/lib/go-1.20/bin:$${PATH}` may be better to ensure using Go 1.20 if available.)